charts/fleet/Chart.yaml

```diff
@@ -4,11 +4,11 @@ name: fleet
 keywords:
   - fleet
   - osquery
-version: v6.4.3
+version: v6.4.3a
```

Comment:
@rfairburn Adding this since I can't just increment the patch version as it will conflict with later versions.